R/check_code.R

```diff
@@ -18,7 +21,8 @@
 #' 2. Make a "-solution" code chunk for the exercise to be checked in a learnr
 #' document. There is no need to supply a solution argument for
 #' \code{check_code()} if you call it from the "-check" chunk of the same
-#' exercise. Likewise, there is no need to supply a user argument when you call
+#' exercise. Likewise, there is no need to supply a student submitted code
+#' argument when you call
```

Comment:
`call it.`?

Reply:
the documentation continues on the next line, I just didn't reflow the docstring to make a smaller diff

Reply:
reflow the docs since it can be done easily within rstudio